src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -3,6 +3,24 @@
 open System
 open System.Diagnostics
 
+
+[<AutoOpen>]
+module Prelude =
+    /// obj.ReferenceEquals
+    let inline (==) a b = obj.ReferenceEquals(a, b)
+    /// LanguagePrimitives.PhysicalEquality
+    let inline (===) a b = LanguagePrimitives.PhysicalEquality a b
+    let inline debug msg = Printf.kprintf Debug.WriteLine msg
+    let inline fail msg = Printf.kprintf Debug.Fail msg
+    let inline isNull v = match v with | null -> true | _ -> false
+    let inline isNotNull v = v |> (not << isNull)
+    let inline dispose (disposable:#IDisposable) = disposable.Dispose ()
+    let inline (+=) (a1:'a byref) (a2:'a) = a1 <- a1 + a2
```

Comment:
I don't see the point of `+=` and `-=`. I prefer explicit mutation to some uncommon assignment constructs that I have to look up.